Dockerfile

```diff
@@ -0,0 +1,22 @@
+# We need a lightweight OS and Node preinstalled
+FROM node:9.11-alpine
+MAINTAINER stockerl@unternehmertum.de
+
+RUN npm install webpack webpack-cli
```

Comment:
because we don't have them in package.json (to at least install them with project scope) and this does not have the -g (global flag), I'm surprised that webpack works at all. Maybe project works currently, because it gets a version from a precompiled dist/ folder? Can we add dist/ to .dockerignore? But at least should throw errors. I think correct should be

`npm install -g webpack webpack-cli`

Reply:
Hmm, I don't have any dist/folder locally. 

I assume the install runs as atm root (perhaps we should address this ;) ) and therefore it gets installed globally?